knack/experimental.py

```diff
@@ -7,6 +7,7 @@
 
 _EXPERIMENTAL_TAG = '[Experimental]'
 _experimental_kwarg = 'experimental_info'
+EXPERIMENTAL_MESSAGE = "{} is experimental and under development."
```

Comment:
should we let customer know that experimental commands may be changed/removed in a future release?

Reply:
Hi Zunli!

Let me chime in a bit...

We actually looped in our researcher and UX writers to rephrase the messaging. Their suggestion is that _"under development_" is a more succinct expression

Also @jiasli  -- a super minor grammar call out (which I just realized)

Can we add an "is" to the sentence? --->

"{} is experimental and is under development." 

Seems to be more grammatically accurate

I hope I'm not too late with the comment...

Reply:
This message won't be seen in Azure CLI in any way. It will be overridden, like in Azure/azure-cli#15730.

> I hope I'm not too late with the comment...

Not too late at all, given @evelyn-ys is concerned about using a module variable. 

> "{} is experimental and is under development."

To me, the sentence structure is `{} is [(experimental) and (under development)]`, but I am open to this. + @dbradish-microsoft to comment.

Reply:
Thanks Delora! @dbradish-microsoft